site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -106,6 +105,19 @@ Some distributions come with Git preinstalled but you'll most likely have to ins
 
 *  Install [Xcode](https://itunes.apple.com/us/app/xcode/id497799835?ls=1&mt=12) to build and run the app on iOS. The minimum required version is 11.0.
 
+#### Additional setup for iOS on m1 macs
+
+1. Follow the [React Native environment setup](https://reactnative.dev/docs/environment-setup) docs until the `cocoapods` point, then stop. 
+2. Specify the correct version of xCode in the terminal: `sudo xcode-select --switch /Applications/Xcode.app`
+3. With m1 mac you may have to install cocoapods using a Rosetta terminal. Follow step 1 from [this blog post](https://www.courier.com/blog/tips-and-tricks-to-setup-your-apple-m1-for-development/) to install a Rosetta terminal.
+4. In the Rosetta terminal, change to the `mattermost-mobile/ios` directory and run:
```

Comment:
is rosetta really needed for this ?

Reply:
Honestly I'm not sure. I was following instructions on getting this all to work, so I don't know other than "it works for me". Not sure how I would test it, because my machine is no longer a clean slate like it was before I started this. :/

Reply:
yeah I guess we can leave this like this, I don't have an M1 so I wouldn't really know